.github/workflows/azure-static-web-apps-orange-rock-0d20a680f.yml

```diff
@@ -0,0 +1,45 @@
+name: Azure Static Web Apps CI/CD
+
+on:

```

Comment:
Is this something we really want to trigger on each push/pull request? Maybe on demand (`worlfkow_dispatch:`)?

Reply:
This file is actually automatically generated when you create the Azure Static Webapps. Azure Static Webapps expects a repo that only has a web interface, so it makes sense in this case: you push your web app, it deploys it. In our case it doesn't make sense. We can update this file (and use what you suggest) but each time we re-create a Static Webapps, it will regenerate the same file.

Reply:
I would guess that this is just a sample that static web apps generated for people to take and customize?

I can probably guarantee that not every single static web app wants to deploy on every push and pull request :)

I would think, too, that the resource group/region/etc may change depending on where you want to deploy it.

I think for superheroes it does not make sense to do this on every push and pull request. Probably on-demand, where the parameters (secrets/etc) can be supplied when it's invoked?

Reply:
Iv' removed it

Reply:
What did you remove? Your merge still has the on push/pull request.